### PR TITLE
Mentioned about "nvm install" in readme file

### LINKED ISSUE #527: Mention about n and nvm in readme.
Will assign myself once priority work in done.

### PR DESCRIPTION
**Pull request by amrutsabale**
Fixes #527

@yedhink _a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,14 @@ Rails application built with opinions of BigBinary style of working.
 
 ## Local Development Setup
 
-Install the latest [Node.js](https://nodejs.org) version.
+Clone this repo by running `git clone https://github.com/bigbinary/wheel.git`.
+
+Install the [Node.js](https://nodejs.org) version `v16.4.2` which we have specified inside the `.node-version` file.
+
+```
```
Comment (yedhink) [inline README.md:12]: Please specify syntax like we are doing in the books.
```suggestion
```bash
```

File: README.md [hunk 2]
```diff
@@ -5,7 +5,14 @@ Rails application built with opinions of BigBinary style of working.
...
+nvm install
+```
+
 Make sure that [npm](https://www.npmjs.com/) is installed with it as well.
```
Comment (yedhink) [inline README.md:16]: We use yarn. You can change `npm` usages into `yarn`.

**Review by yedhink — COMMENTED** [review]
@amrutsabale _a

I have added some comments. Please go through them.

**Comment by amrutsabale** [comment]
@yedhink  _a  
Added review changes.

**Pull request merged**